tools/checked-c-convert/CheckedCConvert.cpp

```
@@ -336,9 +343,16 @@ int main(int argc, const char **argv) {
   else
     llvm_unreachable("No action");
 
+  if (!Info.link()) {
+    errs() << "Linking failed!\n";
+    return 0;
```

You should probably return an error code.